.github/workflows/reusable-module-prep-release.yml

```diff
@@ -221,36 +221,57 @@ jobs:
             echo "EOF"
           } >> "$GITHUB_OUTPUT"
 
-      # create the release branch, commit the changes, and create pull request
-      - name: Create Release Pull Request
+      # create the release branch, commit the changes with the github-actions[bot]
+      - name: Create and commit to release branch
+        id: push-changes
         env:
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          OLD_VERSION: ${{ steps.old.outputs.version }}
+          NEW_VERSION: ${{ steps.new.outputs.version }}
+        run: |
+          git config user.name "github-actions[bot]"
+          git config user.email "41898282+github-actions[bot]@users.noreply.github.com"
+
+          RELEASE_BRANCH="release/${NEW_VERSION}"
+          git checkout -b "$RELEASE_BRANCH"
+          git add .
+          git commit -m "🤖 prep for release - ${OLD_VERSION} → ${NEW_VERSION}"
+          git push origin "$RELEASE_BRANCH"
+
+      - name: Create Pull Request to target branch
+        id: create_pr
+        env:
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           OLD_VERSION: ${{ steps.old.outputs.version }}
           NEW_VERSION: ${{ steps.new.outputs.version }}
           POST_SET_VERSION_MESSAGE: ${{ steps.run-post-set-version.outputs.message }}
           BUILD_OUTPUT_MESSAGE: ${{ steps.run-build.outputs.message }}
           I18N_OUTPUT_MESSAGE: ${{ steps.run-i18n.outputs.message }}
           MERGED_PRS: ${{ steps.merged-prs.outputs.prs }}
-        id: create_pr
-        uses: peter-evans/create-pull-request@271a8d0340265f705b14b6d32b9829c1cb33d45e  # v7.0.8
-        with:
-          token: ${{ github.token }}
-          base: ${{ inputs.module-branch }}
-          branch: release/${{ steps.new.outputs.version }}
-          title: "Automated ${{ inputs.level }} release: ${{ steps.new.outputs.version }}"
-          body: |
-            ## Changes in this pull request
-
-            - Bumped version from `${OLD_VERSION}` to `${NEW_VERSION}`
-            - ${POST_SET_VERSION_MESSAGE}
-            - ${BUILD_OUTPUT_MESSAGE}
-            - ${I18N_OUTPUT_MESSAGE}
-
-            ## Merged PRs since last release
+          RELEASE_LEVEL: ${{ inputs.level }}
+          PLUGIN_TARGET_BRANCH: ${{ inputs.plugin-target-branch }}
```

Comment:
Actually, this is incorrect. The linting workflow is correctly flagging that this is not an input for the workflow.

Reply:
Good catch, not sure how this worked in the solutions example, I guess it was empty and defaulted to the default branch.